lib/viewmodel/club_detail/club_detail_viewmodel.dart

```diff
@@ -14,6 +16,10 @@ class ClubDetailViewModel with ChangeNotifierEx {
   List<Member>? members;
   List<Member>? admins;
   List<Activity>? activities;
+  ActivitySummary? activitySummary;
```

Comment:
I think it's better to work with `ActivitySummary? _activitySummary;` and then create a `ActivitySummary? get activitySummary => _activitySummary;`

Reply:
Would do this also for the other properties